pyccl/__init__.py

```diff
@@ -150,6 +150,17 @@
 from .pyutils import debug_mode, resample_array
 
 # Deprecated & Renamed modules
+def __getattr__(name):
+    rename = {"cls": "cells"}
+    if name in rename:
+        from .errors import CCLDeprecationWarning
+        import warnings
+        warnings.warn(f"Module {name} has been renamed to {rename[name]}.",
+                      CCLDeprecationWarning)
+        name = rename[name]
+        return eval(name)
```

Comment:
These lines are uncovered, can you add a test at the top of `test_cls.py`?

Also, can you rename `test_cls` to `test_cells` (and all occurrences of the word `cls` within)?
I found another mention of `cls`: `tests/test_pk2d.py:def test_pk2d_cls()`.
And in `benchmarks/test_cls.py`.